css/parser/tests/errors/important/basic/output.stderr

```diff
@@ -0,0 +1,6 @@
+error: Expected important
+ --> $DIR/tests/errors/important/basic/input.css:2:20
+  |
+2 |     color: white !!important;
```

Comment:
This seems not ideal. I think it span should include two bangs.

Reply:
Do you mean?
```
color: white !!important;
             ^^^^^^^^^^^
```
  
First `!` is valid here, maybe better show only `!important`?

Reply:
Updates, I think we should output `Expected !important`, because `important` is just word